api/mapping/templates/mapping/scanreportfield_list.html

```diff
@@ -27,9 +27,12 @@ <h2>Fields</h2>
                 <th scope="col">Field</th>
                 <th scope="col">Data Type</th>
 		        <th scope="col">Classification System</th>
+
+	        <!---
```

Comment:
Do we want to remove these columns?

Reply:
we dont use them any more, but i'll remove and leave for another PR